examples/field_formats_example/public/app.tsx

```diff
@@ -168,7 +169,7 @@ const CreatingCustomFieldFormatEditor: React.FC<{ deps: Deps }> = (props) => {
 
 export const App: React.FC<{ deps: Deps }> = (props) => {
   return (
-    <EuiProvider>
+    <EuiProvider theme={EuiThemeAmsterdam}>
```

Comment:
Same here
<img width="1167" alt="Screenshot 2024-12-19 at 14 55 13" src="https://github.com/user-attachments/assets/f43afe42-fab5-4d00-9fa7-60cdf698d206" />

Reply:
Thanks @jughosta, we're taking a look now to see what's going on.